docs/source/tutorials/how_to_model_immunity.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<ul><li>fix LaTeX</li><li>adjust to how the immunity_resistance_factor works now</li><li>refer to the multiple virus strain notebook (or copy from there) for an explanation of the contagiousness factor</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/covid-19-impact-lab/sid/pull/144/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>